sormas-backend/src/main/java/de/symeda/sormas/backend/labmessage/LabMessageFacadeEjb.java

```diff
@@ -126,6 +125,7 @@ LabMessage fromDto(@NotNull LabMessageDto source, LabMessage target, boolean che
 			target.setTestReports(testReports);
 		}
 		target.setReportId(source.getReportId());
+		target.setSampleOverallTestResult(source.getSampleOverallTestResult());
```

Comment:
Please include this into the [LabMessageFacadeEjbMappingTest](https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/sormas-backend/src/test/java/de/symeda/sormas/backend/labmessage/LabMessageFacadeEjbMappingTest.java).